### PR TITLE
(#1152) Remove some more static Matcher asserts

### PR DESCRIPTION
**Pull request by victornoel**
This is for #1152.

I created the new todo for continuing into the `package-info.java` file in tests since it made more sense.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1165](https://github.com/yegor256/cactoos/issues/1165) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1165](https://github.com/yegor256/cactoos/issues/1165) is assigned to @scristalli[/z](https://www.0crat.com/u/scristalli), here is [why](https://www.0crat.com/footprint/C63314D6Z/9e46435a-8143-4876-9ab2-2c62e5247506); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

File: src/test/java/org/cactoos/io/HeadInputStreamTest.java [hunk 1]
```diff
@@ -91,10 +90,10 @@ public void testAvailableLessThanTotal() throws Exception {
             new InputOf("testAvailableLessThanTotal").stream(),
             5
         );
-        MatcherAssert.assertThat(
-            "Count of available bytes is incorrect",
+        new Assertion<>(
+            "Must count available bytes",
```
Comment (scristalli) [inline src/test/java/org/cactoos/io/HeadInputStreamTest.java]: @victornoel minor comment: should we always start with a lower case letter for consistency? `must count available bytes`

File: src/test/java/org/cactoos/io/HeadInputTest.java [hunk 2]
```diff
@@ -39,30 +39,30 @@
...
+        ).affirm();
     }
 
     @Test
     public void readsHeadOfShorterInput() throws Exception {
         final String input = "readsHeadOfShorterInput";
-        MatcherAssert.assertThat(
-            "HeadInput incorrectly limited a number of read bytes",
+        new Assertion<>(
+            "must limit a number of read bytes",
```
Comment (scristalli) [inline src/test/java/org/cactoos/io/HeadInputTest.java]: @victornoel minor comment: check alignment with message on line 43. Also I think the semantics could be improved in both cases for these messages (they're a little generic on the expected result for my taste).

File: src/test/java/org/cactoos/io/HeadInputTest.java [hunk 3]
```diff
@@ -39,30 +39,30 @@
 
     @Test
     public void readsHeadOfLongerInput() throws Exception {
-        MatcherAssert.assertThat(
-            "HeadInput couldn't limit a number of read bytes",
+        new Assertion<>(
+            "must limit the number of read bytes",
```
Comment (scristalli) [inline src/test/java/org/cactoos/io/HeadInputTest.java]: @victornoel minor comment: check alignment with message on line 58. Also I think the semantics could be improved in both cases for these messages (they're a little generic on the expected result for my taste).

**Comment by victornoel** [comment]
@scristalli good points, tell me if it's good like that, in the end I mostly rewrote the second test description to better match the intent.

**Comment by scristalli** [comment]
@llorllale I approve the changes

**Comment by scristalli** [comment]
@llorllale is it possible to either merge or refuse this PR?

**Comment by 0crat** [comment]
This pull request [#1165](https://github.com/yegor256/cactoos/issues/1165) is assigned to @vzurauskas[/z](https://www.0crat.com/u/vzurauskas), here is [why](https://www.0crat.com/footprint/C63314D6Z/1cbb15bd-5327-4f39-a4d8-a54c471a3684); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by vzurauskas** [comment]
@llorllale Another job assigned to me which was done by @scristalli

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) everybody who has role `REV` is banned at [#1165](https://github.com/yegor256/cactoos/issues/1165); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) everybody who has role `REV` is banned at [#1165](https://github.com/yegor256/cactoos/issues/1165); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) everybody who has role `REV` is banned at [#1165](https://github.com/yegor256/cactoos/issues/1165); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) everybody who has role `REV` is banned at [#1165](https://github.com/yegor256/cactoos/issues/1165); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) everybody who has role `REV` is banned at [#1165](https://github.com/yegor256/cactoos/issues/1165); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
This pull request [#1165](https://github.com/yegor256/cactoos/issues/1165) is assigned to @fabriciofx[/z](https://www.0crat.com/u/fabriciofx), here is [why](https://www.0crat.com/footprint/C63314D6Z/4da2b22a-f8d8-4634-b255-040178ee39ec); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by victornoel** [comment]
@fabriciofx @paulodamaso I've rebased on master for the build to pass

**Comment by paulodamaso** [comment]
@rultor merge

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) all `QA`s are on vacation. Please handle that.

**Comment by 0crat** [comment]
The job [#1165](https://github.com/yegor256/cactoos/issues/1165) is now out of scope

**Comment by 0crat** [comment]
Job `gh:yegor256/cactoos#1165` doesn't exist in WBS, can't create order

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Pull request merged**